pkg/infra/process.go

```diff
@@ -3,15 +3,23 @@ package infra
 import (
 	"fmt"
 	"os"
-	"strconv"
 	"time"
 
 	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
+	"gopkg.in/alecthomas/kingpin.v2"
 )
 
 const loglevel = "TAPE_LOGLEVEL"
 
+var (
+	app = kingpin.New("tape", "A tiny, handy tool to test performance of a deployed Fabric network")
```

Comment:
will those comments show in -h ? if yes, suggest keep same with description in document.

Reply:
Yeah, the comment will show in --help.
```
./tape --help
usage: tape [<flags>] <command> [<args> ...]

**A tiny, handy tool to test performance of a deployed Fabric network**

Flags:
  --help  Show context-sensitive help (also try --help-long and --help-man).

Commands:
  help [<command>...]
    Show help.

  run* [<flags>]
    Start the tape program
```
I did not find a suitable description sentence at the time, so I gathered one sentence from "Why tape".
Use "A simple traffic generator for Hyperledger Fabric" as the comment?

Reply:
CC: @guoger  and @stone-ch .
I just checked with README, well, the #what and #why focusing on what tape not. but what's tape? Keep it simple just `A performance test tool for Hyperledger Fabric` ?